demos/marshaller/index.html

```diff
@@ -48,6 +48,9 @@
                 marshaller = new Marshaller()
                     .target("marshaller")
                     .ddlUrl(newUrl.constructUrl())
+                    .on("commsError", function (source, error) {
+                        alert("Pah Humbug:\n" + source + "\n" + error);
+                    }, true)
```

Comment:
@mlzummo This is an example of how to trap the error at the HTML/Graph/Tabbed marshaller level.
Question:  Currently if the user does not override the commsError method it will default to an Alert - is this what we want?